doc/guides/exact-integration.rst

```diff
@@ -0,0 +1,194 @@
+Integrating neural models using exact integration 
+=================================================
+
+The simple integrate-and fire model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the simple integrate-and-fire model the voltage :math:`V` is given as a solution of the equation:
+
+.. math::
+    C\frac{dV}{dt}=I.
+
+This is just the derivate of the law of capacitance :math:`Q=CV`. When an input current is applied, the membrane voltage increases with time until it reaches a constant threshold :math:`V_{\text{th}}`, at which point a delta function spike occurs.
+
+A shortcoming of the simple integrate-and-fire model is that it implements no time-dependent memory. If the model receives a below-threshold signal at some time, it will retain that voltage boost until it fires again. This characteristic is not in line with observed neuronal behavior.
+
+The leaky integrate-and fire model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the leaky integrate-and-fire model, the memory problem is solved by adding a "leak" term :math:`\frac{-1}{R}V` (:math:`R` is the resistance and :math:`\tau=RC`) to the membrane potential:
+
+.. math::
+    \begin{equation}
+        \text{(1) } \quad \frac{dV}{dt}=\frac{-1}{\tau}V+\frac{1}{C}I.
+    \end{equation}
+
+This reflects the diffusion of ions that occurs through the membrane when some equilibrium is not reached in the cell.
+
+Solving a  homogeneous linear differential equation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To solve :math:`\text{(1)}` we start by looking at a simpler differential equation:
```

Comment:
Can we use a proper cross-reference here instead of inserting an explicit reference? See https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-math-numref.

Reply:
If I use the cross-reference the equation number will be (3), and the number is shown above the equation. See print of screen below. Is this ok?

![image](https://user-images.githubusercontent.com/21308850/82201175-3d8e3100-9900-11ea-83f2-f840775a2c16.png)